packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts

```diff
@@ -66,6 +66,7 @@ export class OdspDocumentServiceFactoryCore implements IDocumentServiceFactory {
             siteUrl: odspResolvedUrl.siteUrl,
             filePath,
             filename: odspResolvedUrl.fileName,
+            containerPackageName: odspResolvedUrl.codeHint?.containerPackageName,
```

Comment:
hmm this seems weird in terms of naming and potentially being optional, is there a way we can know this is always defined?

Reply:
We are  having a assert in create new file path. On resolved URL this is optional because it may not be there in load path.
But for create new it would always be there.